lib/generators/rolemodel/saas/devise/templates/app/views/devise/passwords/edit.html.slim

```diff
@@ -0,0 +1,20 @@
+h2 Change your password
+= simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f|
```

Comment:
Relying on simple_form here. I'd like to make that default for a lot of reasons, however, we need a way to ensure it is in the generator.

Or maybe instead of using the NM specific Devise generated files we rely on the ones that Devise generates by default